scripts/upgrade

```diff
@@ -32,6 +32,14 @@ upgrade_type=$(ynh_check_app_version_changed)
 #=================================================
 ynh_script_progression --message="Ensuring downward compatibility..." --weight=1
 
+# If jellyfin doesn't belong to render group, add it
+if [ ! $(grep "jellyfin:x:[0-9]*:render" /etc/group) ]
```

Comment:
From what I see on my server, the syntax of `/etc/group` is the other way around: `$group:x:[0-9]:$user`.

Reply:
What about:
`exec_as $app groups | grep -w render`
or
`sudo -u $app groups | grep -w render`